src/structures/api/SummonerChallenge.ts

```diff
@@ -131,6 +131,14 @@ export class ChallengePreferences {
    * The IDs of the challenges this summoner has put on display.
    */
   readonly displayedChallengeIds: number[];
+  /**
+   * Type of the border the summoner has currently selected. "1" is level border, "2" is highest rank border.
+   */
+  readonly crestBorder: string;
```

Comment:
```suggestion
  readonly crestBorder: 'level' | 'ranked';
```

Reply:
Yep, left some suggestions to better explain what I meant.

Reply:
Ok, I will do this and set the fallback value to the string, just in case more is added